src/components/MindmapWrapper.vue

```diff
@@ -476,13 +463,8 @@ export default {
             }),
           mounted() {
             this.$nextTick(() => {
-              const width = this.$el.offsetWidth;
-              const height = this.$el.offsetHeight;
-              // node.width = width; // Store the computed size in your node's data
-              // node.height = height; // Store the computed size in your node's data
-              container.attr("width", width).attr("height", height);
+              // TODO: need a function to resize the node
             });
```

Comment:
@dockabernathy01 - Is this function just empty now?

Reply:
Currently, there is no need to resize the node.

Reply:
that's fine - please remove the function if its empty @dockabernathy01

Reply:
@dacoaster, I have updated the code.
Currently it's not used but I think it will be used down the road.